onnxruntime/core/framework/sequential_executor.cc

```diff
@@ -112,7 +112,17 @@ Status SequentialExecutor::Execute(const SessionState& session_state,
 
       kernel_begin_time = session_state.Profiler().StartTime();
     }
-    ORT_RETURN_IF_ERROR(p_op_kernel->Compute(&op_kernel_context));
+
+    const auto& compute_status = p_op_kernel->Compute(&op_kernel_context);
+    if (!compute_status.IsOK()) {
+      std::string msg_string =
```

Comment:
it's better to use ostringstream when you're constructing a string from multiple strings. the + operators will make a copy. even if it doesn't matter so much in this case since we're doing it in the error scenario, it's a good habit.

Reply:
Fixed